broadcasting/07_RTSP_Architecture.md

```diff
@@ -0,0 +1,38 @@
+# Architecture
```

Comment:
I believe that this chapter is about to be rewritten so that to emphasize the changes between that architecture and the RTMP to HLS architecture, isn't it/

Reply:
I added some references to the RTMP architecture.